synfig-studio/src/gui/states/state_bone.cpp

```diff
@@ -73,9 +73,10 @@ using namespace synfigapp;
 	name->set_size_request(px)
 #endif
```

Comment:
https://developer.gnome.org/gtkmm/3.24/classGtk_1_1Alignment.html#details

> Use Widget alignment and margin properties instead.

Can you update this part also, please?